src/picongpu/include/plugins/PhaseSpace/PhaseSpaceFunctors.hpp

```diff
@@ -104,8 +104,8 @@ namespace picongpu
             int p_bin = int( rel_bin * float_X(num_pbins) );
 
             /* out-of-range bins back to min/max */
-            p_bin >= 0 ? /* do not change p_bin */ : p_bin=0;
-            p_bin < num_pbins ? /* do not change p_bin */ : p_bin=num_pbins-1;
+            p_bin >= 0 ? p_bin /* do not change p_bin */ : p_bin=0;
+            p_bin < num_pbins ? p_bin/* do not change p_bin */ : p_bin=num_pbins-1;
 
```

Comment:
I check this explicit code an changed it to 
```C++
 if(p_bin < 0)
    p_bin=0;
 if(p_bin >= num_pbins) p_bin=num_pbins-1;
```

For both cases the PTX code is equal , this means we can change the tenary operator.